code/__HELPERS/pronouns.dm

```diff
@@ -1,5 +1,12 @@
+#define GET_TARGET_PRONOUN(target, pronoun) call(target, ALL_PRONOUNS[pronoun])(null)
+#define GET_GENERIC_PRONOUN(pronoun) call(/datum/, ALL_PRONOUNS[pronoun])(null)
+/// Only call this in the context of REPLACE_PRONOUNS
+#define PRONOUN_REGEX regex("%PRONOUN(_(they|They|their|Their|theirs|Theirs|them|Them|have|are|were|do|theyve|Theyve|theyre|Theyre|s|es))")
```

Comment:
Why have this a define if it's only used in one place?

Just have `var/static/regex/pronoun_regex = regex(blah)`

Reply:
If we don't make a new regex whenever the verb is called, it'll keep the information from the old regex for the first run, and we'll end up calling ReplaceText in possible cases where there is no text that needs to be replaced.

Reply:
Oh, then we can just not make it static, then?

Reply:
Sure! I just made it a macro to make it prominent for future debugging and so I didn't have to stick the ugly regex into the middle of the rest of the code